google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -812,7 +812,10 @@ private void appendLoop() {
         this.streamConnection.send(originalRequestBuilder.build());
       }
     }
+    cleanup();
```

Comment:
Why we need to call it here?

Reply:
This cleanup function is shared by the code at the end of the appendLoop() and when an uncaught exception is thrown. In both cases the append thread needs to be terminated, and the inflight requests cleaned up.